README.mkdn

```diff
@@ -47,6 +47,15 @@ Add the gem to your Gemfile:
 gem 'thredded', '~> 0.5.1'
 ```
 
+Add `user.rb` file into modle for initialize. Please be sure you have current_user method in your user.rb.
```

Comment:
Note that the `current_user` method should be defined in the ApplicationController, not in the model